src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.*;
```

Comment:
Please refer to the [coding standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements), "imported classes should always be listed explicitly" :)